crates/notedeck_columns/src/ui/note/custom_zap.rs

```diff
@@ -397,11 +401,11 @@ impl Display for ZapSelectionButton {
             ZapSelectionButton::First => write!(f, "69"),
             ZapSelectionButton::Second => write!(f, "100"),
             ZapSelectionButton::Third => write!(f, "420"),
-            ZapSelectionButton::Fourth => write!(f, "5K"),
-            ZapSelectionButton::Fifth => write!(f, "10K"),
-            ZapSelectionButton::Sixth => write!(f, "20K"),
-            ZapSelectionButton::Seventh => write!(f, "50K"),
-            ZapSelectionButton::Eighth => write!(f, "100K"),
+            ZapSelectionButton::Fourth => write!(f, "{}", tr!("5K", "Zap amount button for 5000 sats. Abbreviated because the button is too small to display the full amount.")),
+            ZapSelectionButton::Fifth => write!(f, "{}", tr!("10K", "Zap amount button for 10000 sats. Abbreviated because the button is too small to display the full amount.")),
+            ZapSelectionButton::Sixth => write!(f, "{}", tr!("20K", "Zap amount button for 20000 sats. Abbreviated because the button is too small to display the full amount.")),
+            ZapSelectionButton::Seventh => write!(f, "{}", tr!("50K", "Zap amount button for 50000 sats. Abbreviated because the button is too small to display the full amount.")),
+            ZapSelectionButton::Eighth => write!(f, "{}", tr!("100K", "Zap amount button for 100000 sats. Abbreviated because the button is too small to display the full amount.")),
```

Comment:
These strings should be localized because it's not guaranteed that all languages follow this way of abbreviating numbers.